src/styles/BackgroundGifs.css

```diff
@@ -0,0 +1,51 @@
+.background-gif {
+    position: relative;
+    margin-bottom: -75vh;
+    z-index: 1;
+    background-size: 100%;
+}
+
+#Rain {
+    background-image: url("../images/rain.gif");
+    background-repeat: no-repeat;
+    background-size: cover;
+    filter:blur(5px);
+    width: 103vw;
+    height: 105vh;
+}
+
+#Clouds {
+    background-image: url("../images/clouds.gif");
+    background-size: cover;
+    background-repeat: no-repeat;
+    filter:blur(5px);
+    width: 103vw;
+    height: 105vh;
+}
+
+#Snow {
+    background-image: url("../images/snow.gif");
+    background-size: cover;
+    background-repeat: no-repeat;
+    filter:blur(5px);
+    width: 103vw;
+    height: 105vh;
+}
+
+#Clear {
+    background-image: url("../images/sunny.gif");
+    background-size: cover;
+    background-repeat: no-repeat;
+    filter:blur(5px);
+    width: 103vw;
+    height: 105vh;
+}
+
+#welcome {
+    background-image: url("../images/sunny.gif");
+    background-size: cover;
+    background-repeat: no-repeat;
+    filter:blur(5px);
+    width: 103vw;
+    height: 105vh;
+}
```

Comment:
Theres a lot of repetition in here that there doesnt need to be. Looking at the div structure, you are applying each of these sets of styles defined by the IDs to the `.background-gif` class anyway, so they could be consolidated and the IDs only used for their unique URLs, IE:

```
.background-gif {
    position: relative;
    margin-bottom: -75vh;
    z-index: 1;
    background-size: cover;
    background-repeat: no-repeat;
    filter:blur(5px);
    width: 103vw;
    height: 105vh;
}

#Rain {
  background-image: url("../images/rain.gif");
}

#Clouds {
  background-image: url("../images/clouds.gif");
}

#Snow {
  background-image: url("../images/snow.gif");
}

#Clear {
  background-image: url("../images/sunny.gif");
}

#welcome {
  background-image: url("../images/sunny.gif");
}
```